yt_dlp/extractor/periscope.py

```diff
@@ -31,9 +31,11 @@ def _parse_broadcast_data(self, broadcast, video_id):
             'title': title,
             'timestamp': parse_iso8601(broadcast.get('created_at')),
             'uploader': uploader,
+            'twitter_username': broadcast.get('twitter_username'),
```

Comment:
this is extracted in `TwitterBroadcastIE`, and `twitter_username` is not a valid info_dict field
```suggestion
```

Reply:
where exactly? the user name wasn't returned before this addition. Couldn't see while testing it with live streams

Reply:
https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/extractor/twitter.py#L1620

your branch appears to be outdated

Reply:
Thanks, updated 👍